client/me/account-password/index.jsx

```diff
@@ -140,7 +140,7 @@ const AccountPassword = createReactClass( {
 		const failure = head( this.props.accountPasswordData.getValidationFailures() );
 
 		if ( this.props.accountPasswordData.passwordValidationSuccess() ) {
-			return <FormInputValidation text={ translate( 'Your password can be saved.' ) } />;
+			return <FormInputValidation text={ translate( 'Strong Password.' ) } />;
```

Comment:
How do we ping i18n? Is there any documentation where it talks about this process. This will help me familiarise with it because I'm looking forward to making more of such text update changes in the future. 😉👩‍💻

Reply:
Ah, you can add the tag "Needs i18n review" and that should flag someone down. There may also be information about that process in the FG.